locales/fr.js

```diff
@@ -31,6 +31,12 @@ export default {
   'Email': 'Email',
   'Download': 'Télécharger',
 
+  // AppFooter
+  'Privacy policy': 'Privacy policy',
```

Comment:
Although we can infer what these should be from the pages we're linking to, I'm in the habit of waiting for approved translations. Thanks for leaving these in english in the french file.

I've added lines to our translations doc. They'll get deployed in the new year.